go/tasks/plugins/array/k8s/transformer.go

```diff
@@ -49,6 +50,48 @@ func GetNamespaceForExecution(tCtx core.TaskExecutionContext, namespaceTemplate
 	return namespace
 }
 
+// Initializes a pod from an array job task template with a K8sPod set as the task target.
+// TODO: This should be removed (it duplicates the pod plugin logic) once we improve array job handling
```

Comment:
nit: add a date here... let's see when will we actually get to this... any bets?

Reply:
done, let me know if you have other suggestions :)